mmdet/models/utils/se_layer.py

```diff
@@ -56,3 +57,71 @@ def forward(self, x):
         out = self.conv1(out)
         out = self.conv2(out)
         return x * out
+
+
+class DYReLU(BaseModule):
```

Comment:
Shall we keep using DyReLU? It is consistent with DyHead and DyDCNv2

Reply:
I will change it. The original abbreviation is [DY-ReLU](https://arxiv.org/abs/2003.10027), but the authors also use [DyReLU](https://openaccess.thecvf.com/content/ICCV2021/papers/Dai_Dynamic_DETR_End-to-End_Object_Detection_With_Dynamic_Attention_ICCV_2021_paper.pdf).